source/includes/_contact_interactions.md

```diff
@@ -1,5 +1,5 @@
 ## Get Contact Interactions for an Advertiser
-**Summary:** This endpoint supports integration with CRM products. It returns all interactions (or leads) generated by a campaign for an advertiser.  The response is organized by campaigns with the interactions nested within. Please note that the response also includes the associated contact and its meta data.
+**Summary:** This endpoint supports integration with CRM products. It returns all interactions (or leads) generated by a campaign for an advertiser.  The response is organized by campaigns with the interactions nested within. Please note that the response also includes the associated contact and its meta data. HIPAA Compliant.
```

Comment:
Why is the phrase _HIPAA Compliant._ at the end of this sentence?

Reply:
Jayanthi requested that be added to the end of Contacts, Contact Interactions, CRM Integrations, and LEFSe.

Reply:
I refined the verbiage per your's and Jayanthi's recommendations.